examples/nextjs-live-avatars/pages/index.tsx

```diff
@@ -38,7 +38,7 @@ export default function Page() {
   const roomId = useOverrideRoomId("nextjs-live-avatars");
 
   return (
-    <RoomProvider id={roomId}>
+    <RoomProvider id={roomId} initialPresence={{}}>
```

Comment:
@nvie Is this something we'd like to enforce? Should we default `initialPresence` to `{}` if not set?

Reply:
Ah perfect, thank you so much for this explanation, it makes perfect sense!